rl.py

```diff
@@ -2,13 +2,26 @@
 This is a file that will run a simple rl model.
 """
 
+import expe
```

Comment:
It would be reasonable to use `from exp import Experiment` to avoid having to qualify the name everywhere below